tests/conftest.py

```diff
@@ -39,6 +39,12 @@ class DummyDevice(DefaultGaussian):
     _operation_map["Kerr"] = lambda *x, **y: np.identity(2)
 
 
+@pytest.fixture(autouse=True)
+def set_numpy_seed():
+    np.random.seed(9872653)
```

Comment:
```suggestion
    np.random.seed(1967)
```
Not a serious suggestion, but this is my favourite magic number for PL.